catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -284,6 +285,27 @@ public UpdateResponse versionUpdate(UpdateResponse updateResponse)
     }
 
     /**
+     * Versions deleted {@link Metacard}s.
+     * @param deleteResponse Versions this responses deleted metacards
+     */
+    public void versionDelete(DeleteResponse deleteResponse) {
```

Comment:
`versionDeletedMetacards` might make more sense, since `versionDelete` sounds like you might be deleting something.